HoneyBadgers.ConsoleApp/Program.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using System;
+using HoneyBadgers.Logic;
 
 namespace HoneyBadgers.ConsoleApp
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var data = new Data();
+            data.LoadData(args[0], args[1]);
```

Comment:
użycie args[] sprawia problem przy próbie uruchomienia programu z eksploratora (dwuklik na pliku exe). Może lepiej użyć prostego stringa zamiast args[]?

Reply:
chciałam to zrobić w bardziej dynamiczny sposób, ale możliwe że przekomibnowałam, już zmieniam